.github/workflows/ci.yml

```diff
@@ -61,23 +61,44 @@ jobs:
         with:
           python-version: "${{ matrix.python-version }}"
 
-      - name: "Install Python packages"
+      - name: "Install Imaginary and dependencies"
         run: |
           pip install --upgrade pip setuptools coverage
-          pip install . ./ExampleGame
-          pip list
+          pip install .
+          pip freeze
```

Comment:
Nothing wrong with this, but FYI, https://github.com/twisted/python-info-action.  Just a chance to collect all the info everyone might want and have it available consistently across (GitHub Actions) CI.

Reply:
Cool, yea, I'll give that a spin.